README.md

```diff
@@ -18,11 +19,23 @@ Ultramodern is a reimplementation of much of the core functionality of libultra.
 
 Platform-specific I/O is handled via callbacks that are provided by the project using ultramodern. This includes reading from controllers and playing back audio samples.
 
-Currently, ultramodern depends directly on [RT64](https://github.com/rt64/rt64) for rendering. It may be decoupled in the future to allow using other renderers for targeting different platforms.
+ultramodern expects the user to provide and regiter a graphics renderer. The recommended one is [RT64](https://github.com/rt64/rt64).
 
-### Librecomp
+## Librecomp
 
 Librecomp is a library meant to be used to bridge the gap between code generated by N64Recomp and ultramodern. It provides wrappers to allow recompiled code to call ultramodern. Librecomp also provides some of the remaining libultra functionality that ultramodern doesn't provide, which includes:
+
 * Overlay handling
 * PI DMA (ROM reads)
-* EEPROM, SRAM and Flashram saving (these may be partially moved to ultramodern in the future)
\ No newline at end of file
+* EEPROM, SRAM and Flashram saving (these may be partially moved to ultramodern in the future)
+
+## Building
+
+It is recommended to reference this project on your `CMakeLists.txt` file. Note that this project has been developed with `clang` 15, older versions may not work.
+
+For building locally this project (ie, developing new features for the libraries of this project), the following is recommneded:
+
+```bash
+cmake -B build -G Ninja -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang -DCMAKE_BUILD_TYPE=Debug
+cmake --build build -j $(nproc) --config Debug
```

Comment:
CMake + Ninja uses parallel jobs by default, so `-j $(nproc)` isn't necessary here. I'm also not sure that --config Debug does anything, especially since CMAKE_BUILD_TYPE is set in the configure stage.

Reply:
Oh, I assumed the `--config` did something since it is mentioned on the Zelda build instructions
https://github.com/Zelda64Recomp/Zelda64Recomp/blob/dev/BUILDING.md#5-building-the-project